### PR TITLE
Feat : Add Form R(B) via FormBuilder

### PR DESCRIPTION
**Pull request by john12321**
The first commit 8e44e0853b68f7da10a6eb3328ee6b07cf51bad3 is where most of the implementation action is.
As far as the new layout for FormB goes, no massive changes just an effort to move towards a more GDS-friendly approach by having a single section per page. 

Some follow-up stuff to consider:
When editing the form, the 'shortcut' button appears on all the form pages; it probably shouldn't appear on the last page (as is the case with the current Form B implementation) given that the 'next' navigation gets you there just as quick!
Move more of the text into the JSON file (e.g. important text)
Make the validation schemas a bit more DRY.
Lost the will to live with the test files but they could be more DRY too.

TIS21-5979

### CODE REVIEW

File: components/forms/form-builder/form-r/part-b/formBValidationSchema.ts [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    .test("endDate", "The date is outside the allowed date range", value =>
+      DateUtilities.IsInsideDateRange(value)
+    )
+});
+
+const covid19ValidationSchema = yup.object().shape({
+  selfRateForCovid: yup
+    .string()
+    .nullable()
+    .required("Self-rating your own preformance is required"),
```
Comment (Judge40) [inline components/forms/form-builder/form-r/part-b/formBValidationSchema.ts:97]: Typo in "performance"

File: redux/slices/formBSlice.ts [hunk 2]
```diff
@@ -53,10 +70,14 @@ export const loadFormBList = createAsyncThunk(
 export const loadSavedFormB = createAsyncThunk(
   "formB/fetchFormB",
   async (formId: string) => {
+    const state = store.getState();
+    // Grab the Covid flag status from the feature flags slice to use to deterimine if the Covid section should be displayed and if a default Covid object should be added to the form data
```
Comment (Judge40) [inline redux/slices/formBSlice.ts:74]: Typo in "determine"

**Review by Judge40 — CHANGES_REQUESTED** [review]
I spotted a couple of minor typos, but otherwise LGTM 😵

**Comment by john12321** [comment]
> I spotted a couple of minor typos, but otherwise LGTM 😵

Good spots! Powered by 'Spoons? 🍻

File: redux/slices/formBSlice.ts (hunk 2 above)
Comment (john12321) [reply redux/slices/formBSlice.ts:74]: I quite like deterimine. Defo belongs in the Covid section.

**Review by Judge40 — APPROVED** [review]

**Pull request merged**